app/src/Routes/NewsEvents/NewsEvents.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
+import * as events from './events.json';
+
+const NewsEvent = (event) => (
+    <figure>
+        <figcaption>{event.title}</figcaption>
+        <figcaption>{event.subtitle}</figcaption>
+        {event.location ? (<span>in {event.location}</span>): ""}
+        {event.date ? (<span> {event.date}</span>): ""}
+        <p>{event.description}</p>
+        {event.postscript ? (<p>{event.postscript}</p>) : ""}
+    </figure>
```

Comment:
same comment as w/ the works page - this isn't the right use for `figure` and `figcaption` elements

Reply:
sounds good, fixed